chart/templates/patch_serviceaccount.yaml

```diff
@@ -0,0 +1,64 @@
+{{- include "_config.load" $ }}
+{{- if gt (len .Values.kube.image_pull_secrets) 0 }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+  name: default
```

Comment:
Maybe use a more descriptive name, like `patch-sa`? Same for the RoleBinding below.

Reply:
confirmed